src/test/java/space/obminyashka/items_exchange/exceptionshandling/GlobalExceptionHandlerTest.java

```diff
@@ -94,16 +94,6 @@ void testHandleIllegalIdentifierException() throws Exception {
         assertThat(result.getResolvedException(), is(instanceOf(IllegalIdentifierException.class)));
     }
 
-    @Test
-    @WithMockUser(username = "admin")
-    @DataSet("database_init.yml")
-    void testHandleInvalidDtoException() throws Exception {
```

Comment:
If the user can/couldn't update such fields why do we expect them from the user?